.github/workflows/lint.yml

```diff
@@ -26,7 +26,10 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
This addition has nothing to do with fixing linter errors.

Reply:
This is just to speed up the checkout since we don't need git history.

Reply:
Do you know whether the `reviewdog/action-golangci-lint` action uses the git history? The default is `fetch-depth: 1` so this will make very little difference, with the risk that it will affect the action.

Reply:
Honestly I don't use ReviewDog, but I checked that they also have it on their examples: https://github.com/reviewdog/action-golangci-lint and since we don't have any extra config assumed it would be ok. If you're aware of any incompatibility I can remove it.

Reply:
Removing.